src/GHCMod/Options/Commands.hs

```diff
@@ -0,0 +1,230 @@
+module GHCMod.Options.Commands where
+
+import Options.Applicative
+import Options.Applicative.Types
+import Language.Haskell.GhcMod.Types
+import GHCMod.Options.DocUtils
+import Language.Haskell.GhcMod.Read
+
+type Symbol = String
+type Expr = String
+type Module = String
+type Line = Int
+type Col = Int
+type Point = (Line, Col)
+
+data GhcModCommands =
+    CmdLang
+  | CmdFlag
+  | CmdDebug
+  | CmdBoot
+  | CmdNukeCaches
+  | CmdRoot
+  | CmdLegacyInteractive
+  | CmdModules Bool
+  | CmdDumpSym FilePath
+  | CmdFind Symbol
+  | CmdDoc Module
+  | CmdLint LintOpts FilePath
+  | CmdBrowse BrowseOpts [Module]
+  | CmdDebugComponent [String]
+  | CmdCheck [FilePath]
+  | CmdExpand [FilePath]
+  | CmdInfo FilePath Symbol
+  | CmdType FilePath Point
+  | CmdSplit FilePath Point
+  | CmdSig FilePath Point
+  | CmdAuto FilePath Point
+  | CmdRefine FilePath Point Expr
+
+commandsSpec :: Parser GhcModCommands
+commandsSpec =
```

Comment:
mconcat is even worse ^^;
Point is, I'd much prefer mconcat operator being no more than three characters wide, so that everything is aligned somewhat beautifully. Otherwise, code becomes full of long spans of whole lotta nothing.
I figured that *if* we go to the trouble of redefining fixities, might as well align the code.